src/applications/vaos/containers/TypeOfSleepCarePage.jsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import SchemaForm from 'platform/forms-system/src/js/components/SchemaForm';
+
+import { TYPES_OF_SLEEP_CARE } from '../utils/constants';
+import FormButtons from '../components/FormButtons';
+import {
+  openFormPage,
+  updateFormData,
+  routeToNextAppointmentPage,
+  routeToPreviousAppointmentPage,
+} from '../actions/newAppointment.js';
+import { getFormPageInfo } from '../utils/selectors';
+
+const initialSchema = {
+  type: 'object',
+  required: ['typeOfSleepCareId'],
+  properties: {
+    typeOfSleepCareId: {
+      type: 'string',
+      enum: TYPES_OF_SLEEP_CARE,
```

Comment:
Using this array here would mean that when a user chooses one of the radio buttons, we would set `typeOfSleepCareId` to either `CPAP` or `Home sleep study`. I think we'd want `typeOfSleepCareId` to be either `340` or `143` after the user made their choice, since those are the codes we need to eventually submit to the api to create a request.

My suggestion would be to make `TYPES_OF_SLEEP_CARE` look like this in the constants file:

```js
export const TYPES_OF_SLEEP_CARE = [
  {
    id: '349',
   name: 'CPAP',
  },
  {
    id: '143',
    name: 'Home sleep study',
  }
];
```

Then in this file you can set the ids and labels like:

```
enum: TYPES_OF_CARE.map(care => care.id),
enumNames: TYPES_OF_CARE.map(care => care.name),
```

This would also I think let you get rid of the 349 care type in TYPES_OF_CARE and remove the filter you added for it in TypeOfCareField.jsx.

Reply:
Done